src/pages/markets/instruments/_market-symbols.tsx

```diff
@@ -9,74 +9,74 @@ export type MarketSymbol = {
 export const americas: MarketSymbol[] = [
     {
         src: icons.STOCKSUS500,
-        text: <Localize translate_text="US 500" />,
+        text: <Localize translate_text="_t_US 500_t_" />,
```

Comment:
can we use `TString` here and update `Symbol` component to use `Localize`? 
just a suggestion, we have to do it eventually, thanks :pray:

Reply:
@mohsen-deriv It is done , Please verify  🙏🏽